src/hooks/users/useWorldUsersById.ts

```diff
@@ -1,30 +1,33 @@
-import { useMemo } from "react";
+import { isEqual } from "lodash";
 
 import { User } from "types/User";
 
-import { worldUsersByIdSelector } from "utils/selectors";
+import { WithId } from "utils/id";
+import { worldUsersByIdWithoutLocationSelector } from "utils/selectors";
 
 import { isLoaded } from "hooks/useFirestoreConnect";
 import { useSelector } from "hooks/useSelector";
 
 import { useConnectWorldUsers } from "./useConnectWorldUsers";
 
+const noUsersById: Record<string, WithId<User>> = {};
```

Comment:
This type is misleading. It basically assumes that the empty object has users. (when you try to get noUsersById["randomString"])

Reply:
Yup. Should be `Partial<Record<string, WithId<User>>>`, but this is the whole broken/legacy tech debt aspect of `useWorldUsersById` / `useWorldUsersByIdWorkaround` if you scroll down a bit more in that same file. Which I don't want to touch in this PR as it's going to require changing a LOT of code across the codebase.